MANUAL.md

```diff
@@ -46,34 +46,37 @@ Protobuf-ES consists of three npm packages:
 
 ## How to generate code
 
-Let's install the compiler [@bufbuild/buf], our plugin and runtime library:
+The quickstart below shows a simple example of code generation for a `.proto` file.
 
-```shell
-npm install --save-dev @bufbuild/buf @bufbuild/protoc-gen-es
-npm install @bufbuild/protobuf
-```
+1.  Install the compiler [@bufbuild/buf], plugin, and runtime library:
 
-Create a `buf.gen.yaml` file that looks like this:
+    ```shellsession
+    npm install --save-dev @bufbuild/buf @bufbuild/protoc-gen-es
+    npm install @bufbuild/protobuf
+    ```
 
-```yaml
-# Learn more: https://buf.build/docs/configuration/v2/buf-gen-yaml
-version: v2
-inputs:
-  - directory: proto
-plugins:
-  - local: protoc-gen-es
-    out: src/gen
-    opt: target=ts
-```
+2.  In a new folder, create a `buf.gen.yaml` file that looks like this:
```

Comment:
This file shouldn't be created in a new folder. The `npm install` commands imply that we're already in a new blank project folder, and `buf.gen.yaml` should sit at the root of that project folder.

The complete implied setup would be something like:

```shellsession
mkdir example
cd example
npm init -y
npm install typescript
npx tsc --init
```

Then the install commands.

Should we include this step for users who aren't familiar with the ecosystem? I tend to think so.

Tangential: For shell examples, do we want to include prompts? Then it would be:

```shellsession
$ mkdir example
$ cd example
$ npm init -y
$ npm install typescript
$ npx tsc --init
```

The advantage of leaving _out_ prompts is that the snippet can be copied, pasted, and run. It'll run all commands sequentially.

Reply:
I removed the prompts—on buf.build, we have some code that causes them not to be copied but GitHub apprarently doesn't. I agree on adding the steps too, so put those in.